Resources/public/js/views/ez-fieldeditview.js

```diff
@@ -130,95 +140,74 @@ YUI.add('ez-fieldeditview', function (Y) {
         },
 
         /**
-         * Show the tooltip taking into account distance between it's supposed
-         * position and bottom of the screen
+         * Check if the fieldValue is valid, and if it is, hide de description
          *
-         * @method _showTooltip
+         * @method _checkValidityDescription
          * @protected
          */
-        _showTooltip: function () {
-            var container = this.get('container'),
-                tooltip = container.one(TOOLTIP_SEL),
-                infoIcon = container.one(FIELD_INFO_ICON_SEL),
-                tooltipHeight;
-
-            tooltip.addClass(IS_DISPLAYED_CLASS);
-
-            if (this._tooltipFitsTailUp()) {
-                // making sure, that the default tail state is in place
-                // and removing changes to the tooltip position (if any)
-                if (tooltip.hasClass(TOOLTIP_TAIL_DOWN_CLASS)) {
-                    tooltip.addClass(TOOLTIP_TAIL_UP_CLASS);
-                    tooltip.removeClass(TOOLTIP_TAIL_DOWN_CLASS);
-                    tooltip.setStyle('top', 'auto');
-                }
-            } else {
-                // switching tooltip to the tail-down state
-                tooltip.addClass(TOOLTIP_TAIL_DOWN_CLASS);
-                tooltip.removeClass(TOOLTIP_TAIL_UP_CLASS);
-                tooltipHeight = parseInt(tooltip.getComputedStyle('height'), 10);
-                tooltip.setY(infoIcon.getY() - tooltipHeight);
+        _checkValidityDescription: function () {
+            if (this.isValid()) {
+                this._hideDescription();
             }
-            tooltip.addClass(IS_VISIBLE_CLASS);
-            this._attachedViewEvents.push(tooltip.on('clickoutside', Y.bind(this._handleClickOutside, this)));
         },
 
         /**
-         * Hides the tooltip
+         * Show the field description
          *
-         * @method _hideTooltip
+         * @method _showDescription
          * @protected
          */
-        _hideTooltip: function () {
-            var tooltip = this.get('container').one(TOOLTIP_SEL);
-
-            tooltip.removeClass(IS_VISIBLE_CLASS);
-            tooltip.removeClass(IS_DISPLAYED_CLASS);
-            tooltip.detach('clickoutside');
+        _showDescription: function () {
+            var container = this.get('container');
+            if (this._handleFieldDescriptionVisibility === true) {
```

Comment:
don't need to check against `true`, `_handleFieldDescriptionVisibility` is not supposed to contain anything than a Boolean